source/direct_debit/index.html.md.erb

```diff
@@ -255,96 +133,197 @@ Required arguments:
+
+You can use the following query parameters to split results into pages:
+
+* `display-size`, which has a default of `500`, and must be between `1` and `500`
+
+* `page`, which has a default of `1`
+
+These must be a positive integer. For example, for the following search:
+
+`GET
+/v1/directdebit/mandates?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z`
+
+if there were 1500 mandates in the response, you could get a page of results by using:
+
+`GET
+/v1/directdebit/mandates?from_date=2018-10-01T13:30:00Z&to_date=2018-10-14T15:00:00Z&display-size=500&page=2`
+
+This would return payments 501 to 1000.
+
+## Mandate statuses
 
-<br>
 <div style = "overflow-x:auto;">
   <table style = "width:100%">
     <tr>
     <th nowrap>State</th>
     <th>Description</th>
     </tr>
     <tr>
-    <td nowrap><code>CREATED</code></td>
-    <td>Your service made an API call to create the agreement.</td>
+    <td nowrap><code>created</code></td>
+    <td>Your service made an API call to create the mandate.</td>
     </tr>
-    <td nowrap><code>STARTED</code></td>
-    <td>Your service sent an API request to GOV.UK Pay.</td>
+    <td><code>started</code></td>
+    <td>The user has visited the <code>next_url</code> and landed on the GOV.UK Pay
+page to enter their account information.</td>
     </tr>
-    <td nowrap><code>PENDING</code></td>
+    <td><code>pending</code></td>
     <td>Indicates that account details have been submitted to the
 payment service provider (PSP), but the PSP is not yet able to confirm whether
-the payment is complete.</td>
+the mandate has been successfully set up.</td>
+    </tr>
+    <tr>
+    <td><code>active</code></td>
+    <td>The PSP has confirmed that the mandate has been succesfully set up.</td>
+    </tr>    
+    <tr>
+    <td><code>inactive</code></td>
+    <td>You cannot collect payments from the mandate, for example because the user cancelled it.
+    </tr>    
+    <tr>
+    <td><code>cancelled</code></td>
+    <td>When the user entered their bank account information, they answered <b>No</b> to <b>Are you allowed to authorise Direct Debits on this account?</b>.</td>
+    </tr>   
   </table>
 </div>
 
-The payment reaches one of several finished states.
+## Payment statuses
 
 <br>
 <div style = "overflow-x:auto;">
   <table style = "width:100%">
     <tr>
-    <th nowrap>Finished state</th>
+    <th nowrap>State</th>
     <th>Description</th>
+    </tr>
+    <td nowrap><code>pending</code></td>
+    <td>The payment has been requested, but the PSP is not yet able to confirm whether
+the payment is complete.</td>
     </tr>
     <tr>
-    <td><code>SUCCESS</code></td>
-    <td>Payment succeeded.</td>
+    <td><code>success</code></td>
+    <td>The payment succeeded.</td>
     </tr>
-    <td><code>FAILED</code></td>
-    <td>Payment failed.</td>
+    <td><code>failed</code></td>
+    <td>The payment failed.</td>
     </tr>
-    <td><code>CANCELLED</code></td>
+    <tr>
+    <td><code>cancelled</code></td>
```

Comment:
I’m not actually sure if we have this functionality yet.

Reply:
Removed it :) https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c